charts/kube-bench-adapter/templates/cronjob.yaml

```diff
@@ -23,5 +26,27 @@ spec:
             "-kubebenchImg", "{{ .Values.kubeBench.kubebenchImg }}",
             "-kubeconfig", "{{ .Values.kubeBench.kubeconfig }}",
             ]
+            {{- with .Values.resources }}
+            resources: {{ tpl (toYaml .) $ | nindent 14 }}
+            {{- end }}
+            {{- with .Values.livenessProbe }}
+            livenessProbe: {{ tpl (toYaml .) $ | nindent 14 }}
+            {{- end }}
+            {{- with .Values.readinessProbe }}
+            readinessProbe: {{ tpl (toYaml .) $ | nindent 14 }}
+            {{- end }}
+            securityContext:
+              allowPrivilegeEscalation: false
+              capabilities:
+                drop:
+                - ALL
+              readOnlyRootFilesystem: true
+              runAsGroup: 1000
+              runAsNonRoot: true
+              runAsUser: 1000
```

Comment:
we can use also create a func for security context as well .

Reply:
okay, doing that too

Reply:
donw